Tasks/AppCenterDistributeV3/Tests/L0.ts

```diff
@@ -310,4 +311,6 @@ describe('AppCenterDistribute L0 Suite', function () {
         tr.run();
         assert(tr.succeeded, 'task should have succeeded');
     });
+
+    utilsUnitTests();
```

Comment:
Could you add some description to the PR covering why we were unable to use the `MockTestRunner` for these utils unit tests? I presume it has to do with checking exit codes, but I'm not sure I fully understand. Thanks!